charts/snowplow-iglu-server/values.yaml

```diff
@@ -0,0 +1,118 @@
+# -- Cloud specific bindings (options: aws, gcp)
+cloud: ""
+
+service:
+  # -- Whether to run the post-deploy setup hooks
+  deploySetupHooks: true
+
+  # -- Port to bind and expose the service on
+  port: "8080"
+  image:
+    repository: "snowplow/iglu-server"
+    tag: "0.8.3"
+    # -- Whether the repository is public
+    isRepositoryPublic: true
+  minReplicas: 1
+  maxReplicas: 4
+  targetCPUUtilizationPercentage: 75
+  terminationGracePeriodSeconds: 630
+
+  readinessProbe:
+    initialDelaySeconds: 5
+    periodSeconds: 5
+    timeoutSeconds: 5
+    failureThreshold: 3
+    successThreshold: 2
+
+  resources:
+    limits:
+      cpu: 746m
+      memory: 900Mi
+    requests:
+      cpu: 400m
+      memory: 512Mi
+
+  config:
+    secrets:
+      # -- Lowercase uuidv4 to use as admin apikey of the service (default: auto-generated)
+      superApiKey: ""
+    repoServer:
+      maxConnections: 16384
+      idleTimeout: "65 seconds"
+    database:
+      # -- Can be either 'dummy' (in-memory) or 'postgres'
+      type: "dummy"
+      # -- Postgres database host
+      host: ""
+      # -- Postgres database port
+      port: 5432
+      # -- Postgres database name
+      dbname: ""
+      secrets:
+        username: ""
+        password: ""
+    # -- Whether to allow schema patching
+    patchesAllowed: false
+
+    # -- Optional Base64 encoded config HOCON (note: will not override above settings)
+    hoconBase64: ""
+    # -- Optional JAVA_OPTS inputs for the deployed service
+    javaOpts: ""
```

Comment:
@istreeter I have allowed for the common settings to be configured here - they are passed in as env vars and always take precedence.  As well as allowing for a general HOCON file to be dumped in to extend and control the other default settings and/or define config settings directly using JAVA_OPTS inputs.

Wdyr?

Reply:
I like it.  The implementation is neat, so it is no bother for us to support these multiple ways of configuring options.  I have no idea if open source users would prefer to provide java opts or a custom hocon file to configure the advanced options, but at least this way we offer them either.

Internally, do we have a preference on whether to use `javaOpts` or `hoconBase64`?  Or do you imagine that for our own use all the params we want to configure will have explicit helm config options?

Reply:
I think we will likely use javaOpts OR end up exposing more settings in the chart directly if they become common enough to warrant it.  This is likely to come down to how many settings we need to expose - for Iglu there are a handle needed to configure it for Production.  The Collector has way more options so we might end up with HOCON for that.

In any case we need JAVA_OPTS to be able to configure the JVM appropriately so the user can chose.